Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -227,6 +227,10 @@ open class AVFoundationPlayback: Playback {
     private var canStartAt: Bool {
         return startAt != 0.0 && playbackType == .vod
     }
+    
+    private var canLiveStartTime: Bool {
```

Comment:
What do you think about giving a more descriptive name to this variable? I think the word `time` here does not fit. Something around `canStartLiveAt`, `canLiveStartAtTime` or even `canLiveStartAtGivenTime`.

Reply:
Done